src/util/rocksdb/mod.rs

```diff
@@ -134,11 +135,27 @@ fn check_and_open(
     let existed: Vec<&str> = cfs_list.iter().map(|v| v.as_str()).collect();
     let needed: Vec<&str> = cfs_opts.iter().map(|x| x.cf).collect();
 
+    // Turn "dynamic level size" off for existing column family which was off before.
+    // column families are small, HashMap isn't necessary
+    let (_, cf_decs) = load_latest_options(path, &Env::default(), false)?;
+    let tune_dynamic_level_size =
+        |x: &mut CFOptions| match cf_decs.iter().find(|(cf_name, _)| cf_name == x.cf) {
+            Some((_, ref cf_dec)) => {
+                if !cf_dec.get_level_compaction_dynamic_level_bytes() {
+                    x.options.set_level_compaction_dynamic_level_bytes(false);
+                } else {
+                    x.options.set_level_compaction_dynamic_level_bytes(true);
```

Comment:
You still need to obey the configuration file:
1. if the configuration says that we should disable it, we should disable it anyway
2. if the configuration says that we should enable it, but we didn't enable it before, we should still disable it
3. if the configuration says that we should enable it, and we did enable it before or it is a new instance, we should enable it

Reply:
Is it a good idea to disable it for a existing instance which was enable before?

Reply:
Oh yes, you are right.